src/persist-client/src/examples/persistent_source_example.rs

```diff
@@ -173,6 +182,33 @@ mod api {
             }
         }
     }
+
+    // TODO: Obviously a placeholder, in the real world, there would be timeouts, and maybe a
+    // `Result` return type.
+    //
+    // Also, the updates are not differential in this interface, while they probably should be.
+    // As it is now, we can never retract updates.
+    #[async_trait]
+    pub trait IteratedConsensus<K, V, T>: Clone
```

Comment:
checking my understanding: a real impl of this would be backed by a persist shard (or rather a storage collection), right? as opposed to directly using persist's `Consensus` trait

Reply:
Yes! I'm adding a comment that mentions this.